lib/features/schedule/widgets/schedule_settings.dart

```diff
@@ -290,11 +234,17 @@ class _ScheduleSettingsState extends State<ScheduleSettings> {
     final chips = <Widget>[];
 
     for (final CalendarFormat format in model.calendarFormatPossibles) {
-      chips.add(InputChip(
+      final bool isTwoWeeksView = model.calendarFormat == CalendarFormat.twoWeeks;
+      final bool isCalendarView = model.toggleCalendarView;
```

Comment:
Enlever de la boucle for

Reply:
Aussi, comme j'ai dit en personne, on pourrait mettre tout ça dans le ViewModel et appeler `notifyListeners()` quand la liste est modifiée

Reply:
Pour la boucle for, bien vu. Par contre pour le `notifyListeners` je vais laisser faire.